README.md

```diff
@@ -109,7 +109,8 @@ This module exposes a single function `download` which takes the same arguments
 * **cut_framerates** a list of additional framerates to detect jump cuts at. If None, jump cuts will only be detected at the original framerate of the video (default *None*)
 * **cuts_are_clips** whether or not to turn each contiguous section of each input video into a distinct ouput video (default *False*)
 * **stage** which stage of processing to execute in betweeen downloading + cheap subsampling and costly subsampling (default *"download"*)
-* **optical_flow_detector** Which optical flow detector to use (default *"cv2"*)
+* **optical_flow_detector** Which optical flow detector (can be "cv2" or "raft") to use (default *"cv2"*)
+* **optical_flow_params** Tuple of hyperparameters to modify optical flow detector (default *None*) 
```

Comment:
this is confusing, too many params about optical flow also optical_flow_params is too vague. Let's just choose one direction and go with that. I suggest one dict param: optical_flow_params and putting everything in there - detector, fps, downsample size etc.

also come up with some reasonable defaults for this